app/scripts/lib/app-start.js

```diff
@@ -302,55 +302,63 @@ define(function (require, exports, module) {
 
     initializeAuthenticationBroker () {
       if (! this._authenticationBroker) {
+        let assertionLibrary = this._assertionLibrary;
+        let iframeChannel = this._iframeChannel;
+        let metrics = this._metrics;
+        let oAuthClient = this._oAuthClient;
+        let relier = this._relier;
+        let session = Session;
+        let window = this._window;
```

Comment:
`win` won't be able to use object shorthand, all the others still can though. For example, change https://github.com/mozilla/fxa-content-server/pull/4335/files#diff-b80f22cfd726dfdc2bb7ed3e82dc7264R321 to:
```js
         iframeChannel,
         relier,
         window: win
```

It could be that I'm remembering ghosts of problems that never actually existed, but shadowing `window` and other global objects makes me twitchy.